### PR TITLE
Initial backend

### PR DESCRIPTION
**Pull request by KyoshiNoda**
installed nodemon and have local host 3001 running

added template endpoints for following:
- accounts
- categories
- items
- authentication 
#29

### CODE REVIEW

File: backend/server.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+app.delete('api/categories/:id',(req,res) =>{
+  res.send({response : 'deleted id'});
+});
+
+
+app.get('/api/item/:id',(req,res) =>{
+  res.send('hello')
+});
+
+app.get('/api/items',(req,res) =>{
```
Comment (J-Obog) [inline backend/server.js]: /api/item

File: backend/server.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+app.patch('/api/account',(req,res) =>{
+  res.send({account : 'updated info!'});
+});
+
+app.post('/api/account',(req,res) =>{
+  res.send({response: 'account was made!'});
+});
+
+app.get('/api/categories/:id',(req,res) =>{
```
Comment (J-Obog) [inline backend/server.js]: i think we should do /api/category

File: backend/server.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  res.send({id: 999});
+});
+
+app.get('/api/categories', (req,res) =>{
```
Comment (J-Obog) [inline backend/server.js]: api/category

File: backend/server.js [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+});
+app.post('/api/auth/revoke',(req,res) =>{
+  res.send({response : 'revoked!'});
+});
+
+app.get('/api/account/:id',(req,res) =>{
+  res.send({id : 999});
+});
+
+app.get('/api/accounts', (req,res) =>{
```
Comment (J-Obog) [inline backend/server.js]: we probably won't need an endpoint for retrieving multiple accounts

File: backend/server.js [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+});
+
+app.post('/api/auth/tokens',(req,res) =>{
+  res.send({token : 'J12KL3H1F'})
+});
+app.post('/api/auth/revoke',(req,res) =>{
+  res.send({response : 'revoked!'});
+});
+
+app.get('/api/account/:id',(req,res) =>{
```
Comment (J-Obog) [inline backend/server.js]: api/account, we know the user based on the token

**Comment by J-Obog** [comment]
Looking great! Just had a few comments for naming convention.

File: backend/server.js (hunk 1 above)
Comment (KyoshiNoda) [reply backend/server.js]: fixed!

File: backend/server.js (hunk 2 above)
Comment (KyoshiNoda) [reply backend/server.js]: fixed!

File: backend/server.js (hunk 3 above)
Comment (KyoshiNoda) [reply backend/server.js]: fixed!

File: backend/server.js (hunk 4 above)
Comment (KyoshiNoda) [reply backend/server.js]: fixed!

File: backend/server.js (hunk 5 above)
Comment (KyoshiNoda) [reply backend/server.js]: fixed!

**Review by J-Obog — APPROVED** [review]
LGTM 💲

**Pull request merged**